lib/commands/config/inject.ts

```diff
@@ -76,12 +68,11 @@ export default class ConfigInjectCmd extends Command {
 			ConfigInjectCmd,
 		);
 
-		const { promisify } = await import('util');
-		const umountAsync = promisify((await import('umount')).umount);
+		const { safeUmount } = await import('../../utils/helpers');
 
 		const drive =
 			options.drive || (await getVisuals().drive('Select the device/OS drive'));
-		await umountAsync(drive);
+		await safeUmount(drive);
```

Comment:
If `drive` was a path to an image file like `./balena.img` (as opposed to a path to a block device like `/dev/disk2`), `umountAsync('./balena.img')` would raise an error. This PR adds a check on whether `drive` is something currently mounted before attempting to umount it.